modules/move-etcd-different-disk.adoc

```diff
@@ -159,37 +90,91 @@ metadata:
 spec:
   config:
     ignition:
-      version: 3.4.0
+      version: 3.1.0
+    storage:
+      files:
+        - path: /etc/find-secondary-device
+          mode: 0755
+          contents:
+            source: data:text/plain;charset=utf-8;base64,<encoded_etc_find_secondary_device_script> # <1>
```

Comment:
Looking at the preview, I can see that the callout actually extends beyond what is normally visible to the reader without scrolling. :disappointed: 

I don't think that it's critical that we do anything about that _right now_, but it's worth knowing about.

Reply:
Oh, we can shorten the name to `<encoded_etcd_script>` or similar! I did this later in the review (line 173), but forgot to update this comment.